### PR TITLE
Documentation website

### PR DESCRIPTION
**Pull request by dimamo5**
This PR adds a documentation website to Velocidi Android SDK.

### CODE REVIEW

File: docs/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# abort on errors
+set -e
+
+# build
+yarn run dist
+
+# navigate into the build output directory
+cd temp/.vuepress/dist
+
+# if you are deploying to a custom domain
```
Comment (beatriz) [inline docs/deploy.sh]: Can you better specify these comments? The "if" seems that this is a configuration and we can set up to a custom domain or not, but I don't believe it is the case.

File: docs/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+git init
+git add -A
+
+git config user.email "baco@velocidi.com"
+git config user.name "Baco"
+
+git commit -m 'deploy'
+
+# if you are deploying to https://<USERNAME>.github.io/<REPO>
```
Comment (beatriz) [inline docs/deploy.sh]: Same comment as above.

File: docs/package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "npm": "NO LONGER USED - Please use yarn"
+  },
+  "devDependencies": {
+    "vuepress": "^1.0.2"
+  },
+  "scripts": {
+    "generate-docs": "cd .. && ./gradlew :velocidi-sdk:dokka",
+    "import-docs": "cp -R ../velocidi-sdk/build/docs/velocidi-sdk/ temp && cp -R .vuepress temp",
+    "dev": "yarn run generate-docs && yarn run import-docs && vuepress dev temp",
+    "clean": "rm -R temp && rm -R ../velocidi-sdk/build/docs/",
```
Comment (beatriz) [inline docs/package.json]: As discussed offline:

```suggestion
    "clean": "rm -Rf temp && rm -Rf ../velocidi-sdk/build/docs/",
```

File: docs/deploy.sh (hunk 1 above)
Comment (dimamo5) [reply docs/deploy.sh]: Done in 5b3bbc8

File: docs/deploy.sh (hunk 2 above)
Comment (dimamo5) [reply docs/deploy.sh]: Done in 5b3bbc8

File: docs/package.json (hunk 3 above)
Comment (dimamo5) [reply docs/package.json]: Done in 5b3bbc8

**Comment by dimamo5** [comment]
> LGTM overall just left a couple of small comments. When running, I noticed this:
> Screenshot 2019-07-18 11 52 18
> It would be nice if we could have a bit more spacing between the class name and its description.

I added a small margin in 5b3bbc8. Much better now :)

**Pull request merged**